src/docker-compose.override.yml

```diff
@@ -6,13 +6,13 @@ services:
       - ./var1.env
     environment:
       - ASPNETCORE_ENVIRONMENT=Development
-      - ASPNETCORE_URLS=https://+:65103;http://+:80
+      - ASPNETCORE_URLS=https://+:64103;http://+:80
```

Comment:
Please revert your changes to docker-compose.override.yml.

Reply:
fixed this